sprockets_postgres.py

```diff
@@ -681,7 +680,7 @@ def _on_postgres_error(self,
         LOGGER.error('%s in %s for %s (%s)',
                      exc.__class__.__name__, self.__class__.__name__,
                      metric_name, str(exc).split('\n')[0])
-        if isinstance(exc, ConnectionException):
+        if isinstance(exc, psycopg2.OperationalError):
```

Comment:
Shouldn't this whole branch be removed?

Reply:
Removing the '503' status branch is changing current behavior for connection check, is there a better way to handle?

Reply:
I'm not following @gmr, since we aren't wrapping in `ConnectionException`, I imagine we need keep expectation that these are automatically handled in the `_on_postgres_error`?